tests/queries/0_stateless/01193_metadata_loading.sh

```diff
@@ -19,7 +19,7 @@ threads=10
 count_multiplier=1
 max_time_ms=1000
 
-debug_or_sanitizer_build=`$CLICKHOUSE_CLIENT -q "WITH ((SELECT value FROM system.build_options WHERE name='BUILD_TYPE') AS build, (SELECT value FROM system.build_options WHERE name='CXX_FLAGS') as flags) SELECT build='Debug' OR flags LIKE '%fsanitize%'"`
+debug_or_sanitizer_build=`$CLICKHOUSE_CLIENT -q "WITH ((SELECT value FROM system.build_options WHERE name='BUILD_TYPE') AS build, (SELECT value FROM system.build_options WHERE name='CXX_FLAGS') as flags) SELECT build='Debug' OR flags LIKE '%fsanitize%' OR hasThreadFuzzer()"`
```

Comment:
Do I understand correctly, that thread fuzzer will be enabled for all test configurations? In this case it's probably better just to increase `max_time_ms`.

Reply:
It's unclear how it will be enabled, maybe it will be a separate test run
(but it happens to be useful both in ASan and TSan)

Reply:
It will be perfect if it will be enabled for all tests including stress and integration (and not including performance).
But I'm still not sure - it can slow down tests too much.